### DISCUSSION TITLE
[Slider] fireOnInit: true, resets the labels and ticks to default module settings (labelType: 'number', max: 20)

### DISCUSSION THREAD

**Discussion by mfeherpataky** [fomantic/Fomantic-UI#1233] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Currently the `fireOnInit: true`, fires onChange as expected on the initialisation. However it has an unexpected side-effect on labels and ticks (if enabled). 
While the Thumb behaves and select the values as if these were distributed based on the passed parameters, the labels are always distributed based on the defaults - `max: 20` and `labelType: 'number'`.

The `fireOnInit` is not present in the module documentation, so not sure if it is supported.

## Testcase
https://codepen.io/feher/pen/wvBJwYE

## Version
2.8.2

**Comment by prudho** [comment, marked as answer]
This is more a usage issue IMHO. Calling the slider function into the slider's event callback can disturb it. You should handle the value change like this:

```js
onChange: (value) => {
  $('#slider-input-1').val(value)
},
```

**Comment by mfeherpataky** [comment]
That's a good point... and indeed was at the root cause of the issue.
I think that issue can be closed...
